src/test/regress/expected/multi_having_pushdown.out

```diff
@@ -23,6 +23,7 @@ SELECT l_orderkey, sum(l_extendedprice * l_discount) as revenue
     FROM lineitem_hash
     GROUP BY l_orderkey HAVING sum(l_quantity) > 24
     ORDER BY 2 DESC, 1 ASC LIMIT 3;
+DEBUG:  push down of limit count: 3
```

Comment:
```EXPLAIN``` output doesn't include the limit node's "number" unless we turn on costs which we turn it off in tests. I am thinking about adding a UDF which gives us worker query and use it for testing these kind of stuff. Since we need the limit number, specially for tests with ```OFFSET```, what do you think to keep limit push down ```DEBUG``` output until we have the UDF?